setup.cfg

```diff
@@ -24,6 +24,10 @@ markers =
     installation: Test requires installation of dependencies
     use_sample_data: Run functional tests using real data
     sequential: Run these tests sequentially, see https://github.com/ESMValGroup/ESMValCore/issues/644
+filterwarnings =
+    ignore::UserWarning:iris.*
```

Comment:
Why are you disabling these warnings?

Reply:
we got nothing to do with them and they're bloating the stdout?

Reply:
That's maybe true for the `UserWarning:iris.*`, but the `DeprecationWarning:iris.*` probably should be removed by fixing the code that produces them.

Reply:
iris UserWarning are there to warn you that you're using iris in a dodgy way, e.g. collapsing a coordinate without bounds. The correct way to fix these warnings would be to
1) have a critical look at our tests and see if the test data is similar enough to real data (e.g. does it have bounds?)
2) silence the warnings if they do not make sense in our code (e.g. #1215), because just silencing them in the tests means that users of our software will still get them, but we're not aware of that because we've silenced them in the tests

Reply:
I really don't think stuff like:
```
tests/integration/cmor/_fixes/cmip6/test_cesm2_waccm.py::test_cl_fix_file
  /usr/share/miniconda3/envs/esmvaltool/lib/python3.9/site-packages/iris/fileformats/cf.py:584: UserWarning: Missing CF-netCDF formula term variable 'ps', referenced by netCDF variable 'lev_bnds'
    warnings.warn(
```
see original out in [GA test](https://github.com/ESMValGroup/ESMValCore/runs/3785574476?check_suite_focus=true)
are in any way interesting and/or warrant fixes from us. There's a lot of these types, OK here's one that might be useful:
```
tests/unit/preprocessor/_multimodel/test_multimodel.py::test_ignore_tas_scalar_height_coord
  /usr/share/miniconda3/envs/esmvaltool/lib/python3.9/site-packages/iris/coords.py:1982: UserWarning: Collapsing a non-contiguous coordinate. Metadata may not be fully descriptive for 'height'.
    warnings.warn(msg.format(self.name()))
```
but here too, we've been seeing this sort of warning for years and yet we've not done anything about it. Let's face it guys, I don't think we've actually fixed anything related to a warning until it became a full on raised exception :grin: